### DISCUSSION TITLE
How to setup stage and audience screen

### DISCUSSION THREAD

**Discussion by Shalveen** [ChurchApps/ChurchAppsSupport#331] [FreeShow] [answered]
I have downloaded the software and it is really good. I wish to use it for my church and have the stage facing projector screen display both current slide and next slide lyrics, while the congregation facing screen to show only the current slide screen. Could you please guide me through on how to get this done?

**Comment by vassbo** [comment]
Hi, thank you. 😄
To show the main output for the congregation you just press the output button in the top right, it should become red. If it's not showing up, try changing the output settings.
For stage you can create different layouts under "Stage", and connect from any browser on any device, (see the connections settings, normally like this: "http://[IP]:5511")
Let me know if you have any more questions! 😊

**Reply by paulnatemv** [reply]
It would be good to add NDI support for display and stage screen

**Reply by vassbo** [reply]
@paulnatemv NDI-support is on the todo list. This is currently one of the main features missing, I think. It a bit tricky to implement, but I will see if I can add it soon.

**Comment by jwamoo** [comment]
You're doing an awesome job! Blessings!

**Comment by dantyndall** [comment]
@vassbo, sorry for reviving an old thread... Is there a way to display the Stage Show outside of a web browser, so that video on the Stage display is a little more responsive? I'm thinking of a secondary output that can display a stage layout of my selection.

**Comment by vassbo** [comment, marked as answer]
No worries. On the right, in the "Stage" view. You can enable "Output screen" to get a stage display as a screen output:
![image](https://github.com/ChurchApps/FreeShow/assets/17619496/bf34706d-2ba1-4c60-9f9a-85a3faa9da26)

**Reply by abhishikthraj** [reply]
Sorry, I know it is not relevant here but how do I connect multiple hdmi screens to a laptop with one hdmi port and be able to detect multiple screens?

**Reply by vassbo** [reply]
You can purchase a USB/USB-C dock with multiple HDMI outputs.

**Reply by dantyndall** [reply]
@vassbo Got another question for you. Is it possible changing the stage screen layout on the fly? Sometimes I want to show Lyrics, but other times, I want to duplicate what would be shown on the video screen (like playing a video). I have 2 stage layouts set up:

![image](https://github.com/ChurchApps/FreeShow/assets/73319625/c8be337a-c51d-4070-9632-48bee0dffd8d)

And I'm displaying the stage output on a third monitor like you had told me. I thought maybe "Move all connections to this" would switch the display of the stage layout, but that doesn't seem to do anything.

**Comment by vassbo** [comment]
There is "Move all connections" as you mentioned, and a custom action to change the stage displays.
But that just works for the remote stage display.

You can set up a stage display mirror item to change it like you want: https://freeshow.app/docs/mirrors#stage-display-in-output-window
Or you can just change the physical output.
